src/Deputy/Subscriber/BindAdherentDistrictSubscriber.php

```diff
@@ -28,17 +28,19 @@ public function updateReferentTagWithDistrict(AdherentEvent $event): void
         $adherent = $event->getAdherent();
 
         if ($adherent->isGeocoded()) {
-            $referentTags = $this->districtRepository->findDistrictReferentTagByCoordinates(
+            $districts = $this->districtRepository->findDistrictReferentTagByCoordinates(
```

Comment:
should this method be renamed with something like `findDistrictsByCoordinates($lat, $long)` then ?

Reply:
```suggestion
            $districts = $this->districtRepository->findDistrictReferentTagByCoordinates(
```